src/pages/en/guides/images.mdx

```diff
@@ -29,6 +29,17 @@ import rocket from '../images/rocket.svg';
 <img src={rocket} alt="A rocketship in space."/>
 ```
 
+:::caution
+The official [image integration](#astros-image-integration) will change image imports to return an object.
```

Comment:
```suggestion
When installed, the [official image integration](#astros-image-integration) will change all image imports to return an object, even when you don't use the `<Image />` component.
```
I don't know whether this is a little much, but I do think it's important to say something like "When installed..." behaviour is changed, especially since we're so early on this page!

Reply:
At this point, they haven't been exposed to the `<Image />` component yet, and there's a risk of causing confusion between it and `<img>`. How about this?

```suggestion
When installed, the [official image integration](#astros-image-integration) will change all image imports to return an object, even when you don't use the integration's `<Image />` component.
```

Reply:
Yes, that would be a better version! Still not sure myself if that's toooo much, but I would 100 add "when installed".  Open to thoughts on this!

Reply:
And, still wondering "note/tip" vs caution. Caution seems VERY warn-y, and this is literally the most basic usage of images!  😅 

Could we downplay the aside and _maybe_ add something in Troubleshooting that's optimized for OMG USING BOTH?? and will be a good search hit? Is that a reasonable compromise?

Reply:
I like that! Since it's only a problem when the image integration is installed, that's where the flames go. Perfect.

But, it should be towards the end of the section, not right up front, so it doesn't look overwhelming when you first get there. Explain how the integration works, warning at the end re: normal img tag. 👍

Reply:
Thanks for this discussion - check out the new section!